spec.md

```diff
@@ -2159,6 +2287,50 @@ message NodeGetInfoResponse {
 If the plugin is unable to complete the NodeGetInfo call successfully, it MUST return a non-ok gRPC code in the gRPC status.
 The CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
 
+#### `NodeExpandVolume`
+
+A Node Plugin MUST implement this RPC call if it has `EXPAND_VOLUME` node capability.
+This RPC call allows CO to expand volume on a node.
+
+`NodeExpandVolume` ONLY supports expansion of already node-published or node-staged volumes on the given `volume_path`.
+
+If plugin has `STAGE_UNSTAGE_VOLUME` node capability then:
+* `NodeExpandVolume` MUST be called after successful `NodeStageVolume`.
```

Comment:
This would seem to contradict line 1776 in the offline ControllerExpandVolume OFFLINE instructions.  Those indicate that a volume must be unstaged before it can be expanded, but this line says it must be staged.  Maybe this section seems to be about the online flow.  Does it need to be updated to reflect that NodeExpandVolume also sometimes participates in offline flow?  Or am I just confused?

Reply:
@jdef no you are right. ONLINE and OFFLINE capabilities are entirely about whether you can call `ControllerExpandVolume` while volume is published/available or not.  I thought language of OFFLINE clarified this:

> // OFFLINE indicates that volumes currently published and
    // available on a node SHALL NOT be expanded via
     // ControllerExpandVolume

`NodeExpandVolume` ALWAYS requires that volume is either published or staged. It does not support any kind of OFFLINE expansion on the node.